tests/installation/edit_optional_kernel_cmd_parameters.pm

```diff
@@ -19,29 +19,45 @@ use version_utils qw(is_sle is_leap is_upgrade);
 sub run {
     my ($self) = shift;
 
-    # Verify Installation Settings overview is displayed as starting point
-    assert_screen "installation-settings-overview-loaded";
+    if (check_var('VIDEOMODE', 'text')) {
+        send_key 'alt-c';
+        assert_screen 'inst-overview-options';
+        send_key 'alt-b';
+        assert_screen 'installation-bootloader-config';
+        send_key 'alt-k';
+        assert_screen 'installation-kernel-parameters';
+        send_key 'alt-p';
+        wait_still_screen(1);
```

Comment:
it is recommended avoid introducing artificial sleeps, `wait_screen_change` might help here.